packages/dd-trace/src/appsec/sdk/track_event.js

```diff
@@ -2,12 +2,27 @@
 
 const log = require('../../log')
 const { getRootSpan } = require('./utils')
-const { setUserTags } = require('./set_user')
+const { setUserTagsSdk } = require('./set_user')
 const waf = require('../waf')
 const { keepTrace } = require('../../priority_sampler')
 const addresses = require('../addresses')
 const { ASM } = require('../../standalone/product')
+const telemetryMetrics = require('../../telemetry/metrics')
 
+const appsecMetrics = telemetryMetrics.manager.namespace('appsec')
+
+function increaseSdkEventMetric (eventType, version) {
+  const tags = {
+    event_type: eventType,
+    sdk_version: version
+  }
+
+  appsecMetrics.count('sdk.event', tags).inc(1)
+}
```

Comment:
should we have this in the telemetry file instead of here ? looks a bit out of plac eno ?

Reply:
merged Ilyas changes from master, and removed this code